patches/chromium/can_create_window.patch

```diff
@@ -118,11 +118,18 @@ index a354e100b04423b9793bc17473711fab9ca91c7e..2b03b5b0152678a4f2fa64cabab0dc51
 +                     params.target_url, new_contents);
 +}
 +
- WebContents* WebContentsDelegate::OpenURLFromTab(WebContents* source,
-                                                  const OpenURLParams& params) {
+ WebContents* WebContentsDelegate::OpenURLFromTab(
+-    WebContents* source,
+-    const OpenURLParams& params,
+-    base::OnceCallback<void(NavigationHandle&)> navigation_handle_callback) {
++  WebContents* source,
++  const OpenURLParams& params,
++  base::OnceCallback<void(NavigationHandle&)> navigation_handle_callback) {
    return nullptr;
+ }
+ 
```

Comment:
@deepak1556 this change appears unnecessary/inadvertent

Reply:
hmm interesting, I didn't touch this file apart from using the patch exports script.